Library/Formula/bash.rb

```diff
@@ -6,16 +6,69 @@ class Bash < Formula
 
   stable do
     url "http://ftpmirror.gnu.org/bash/bash-4.3.tar.gz"
-    mirror "https://ftp.gnu.org/gnu/bash/bash-4.3.tar.gz"
```

Comment:
Hmm. I think it's reasonable to keep it as a `mirror` generally. The upstream website says use mirrors when possible, which we do, but doesn't bar using the "canonical" link:

> Bash can be found on the main GNU ftp server: http://ftp.gnu.org/gnu/bash/ (via HTTP) and ftp://ftp.gnu.org/gnu/bash/ (via FTP). It can also be found on the GNU mirrors; please use a mirror if possible.